mappings.xml

```diff
@@ -0,0 +1,131 @@
+<?xml version="1.0"?>
+<mappings>
+  <mapping>
+    <path>~/navigation/position/latitude</path>
+    <sentence>
+      <id>RMC</id>
+      <fieldset type="coordinate">
+        <field>2</field>
+        <field>3</field>
+      </fieldset>
+      <filter type="if">
+        <field>1</field>
+        <value transform="uppercase">A</value>
+      </filter>
+    </sentence>
+  </mapping>
+
+  <mapping>
+    <path>~/navigation/position/longitude</path>
```

Comment:
The `value` of the `navigation.position` should be an object like so
```
{
    "context": "vessels.123456789",
    "updates": [{
        "source": {
            "label": "foobar",
            "type": "NMEA2000",
            "pgn": "129025",
            "src": "160"
        },
        "timestamp": "2014-08-15-16:00:21.860",
        "values": [{
            "path": "navigation.position",
            "value": {
                "longitude": 24.9027157,
                "latitude": 60.0387795
            }
        }]
    }]
}
```

I think that the *value* in the delta message should suffice. The delta should read for a human _here are some updates about this context, here's the source & timestamp and here are the values". Primitive values are primitives, while composite values like position and orientation should be objects.

```
{
    "context": "vessels.123456789",
    "updates": [{
        "source": {
            "label": "foobar",
            "type": "NMEA2000",
            "pgn": "129026",
            "src": "160"
        },
        "timestamp": "2014-08-15-16:00:10.999",
        "values": [{
            "path": "navigation.speedOverGround",
            "value": 3.87
        }, {
            "path": "navigation.courseOverGroundTrue",
            "value": 171.8
        }]
    }]
}
```

For primitive values Multiplexer adds the `.value` key: https://github.com/SignalK/signalk-multiplexer-node/blob/master/lib/delta.js#L31